infrastructure/terraform/environments/blake/main.tf

```diff
@@ -78,4 +79,5 @@ module "deployment" {
   NEXT_PUBLIC_SUPABASE_URL        = local.NEXT_PUBLIC_SUPABASE_URL
   NEXT_PUBLIC_SUPABASE_PUBLIC_KEY = local.NEXT_PUBLIC_SUPABASE_PUBLIC_KEY
   ASSETS_BUCKET_NAME              = local.ASSETS_BUCKET_NAME
+  HOSTNAME                        = local.HOSTNAME
```

Comment:
I think we should also add it here, partially because the `terraform`  linting tool was complaining, and partiallly because of what eve wrote here

https://github.com/pubpub/platform/blob/b31d3b0918ab2e5a7777f7a36e7808bbb4582f7a/infrastructure/terraform/README.md#L30